src/collective/cover/tests/test_searchabletext_indexer.robot

```diff
@@ -24,7 +24,7 @@ ${search_results_number_selector}  xpath=//strong[@id='search-results-number']
 
 *** Test cases ***
 
-Test RichText Tile is Searchable
+Test RichTextTile is Searchable
```

Comment:
It was running two tests instead of one when I use the command 

    $ ./bin/test --layer Robot -t "Test RichText Tile"